src/number_theory/padics/padic_norm.lean

```diff
@@ -372,6 +372,20 @@ begin
     exact_mod_cast split_frac, }
 end
 
+protected lemma div_pow {p : ℕ} [p_prime : fact p.prime] {b k : ℕ} (dvd : p ^ k ∣ b) :
```

Comment:
Is there a lemma that proves `padic_val_nat p (a / b) = padic_val_nat p a - padic_val_nat b` assuming `b ∣ a`? I think the current lemma should use that.

Reply:
Done!